pkg/networkutils/network.go

```diff
@@ -407,6 +407,10 @@ func (n *linuxNetwork) updateHostIptablesRules(vpcCIDRs []string, primaryMAC str
 		if err := n.updateIptablesRules(iptablesConnmarkRules, ipt); err != nil {
 			return err
 		}
+		err = n.cleanOldNatChains(ipt)
```

Comment:
nit: can you add a comment that this cleans up chains from the previous implementation that may have been missed by the reconciliation logic in `updateIptablesRules`?